src/main/scala/io/iohk/ethereum/ledger/BlockRewardCalculator.scala

```diff
@@ -18,6 +18,9 @@ class BlockRewardCalculator(config: MonetaryPolicyConfig) {
   /** Base block reward in the first era */
   val firstEraBlockReward: BigInt = config.firstEraBlockReward
 
+  /** Block reward for miner after Byzantium Fork */
+  val newRewardAfterByzantium: BigInt = BigInt(10).pow(18) * 3
```

Comment:
I think it would be better to write it as number, otherwise scalastyle can complain on magic numbers. Or maybe define it in configuration is some smart way.
One i can think of, is definig new parameter in `MonetaryPolicyConfig` called like 
` first-era-reduced-block-reward` and in `ethereumHf.conf` setting it to correct reduced value, but in `application.conf`  setting it to current value as there is no byzantium fork on etc chain.
What do you think about that ?  It my be overkill, but rewards are highly consensus related thing, and i am a ltittle uneasy hardcoding them like that

Reply:
I've actually tried this solution but then when executing tests the program couldn't see it in some weird way, so I just put it in here, but I could try to investigate further

Reply:
I've work it out, so this reduce block reward is now present in `MonetaryPolicyConfig` as you suggested